builder/rootfs/bin/boot

```diff
@@ -74,6 +88,27 @@ done
 docker build -t deis/slugbuilder /usr/local/src/slugbuilder/
 docker build -t deis/slugrunner /usr/local/src/slugrunner/
 
+function gen_host_keys {
+	# generate the keys, then set them up in etcd if not already set
+	/usr/bin/ssh-keygen -A
```

Comment:
Additionally, the logic seems incorrect to me - it looks like we are generating new keys on *every boot*, then saving them to etcd and re-templating them? This doesn't seem like at all what we want,  unless I'm missing something.

The templating after seems completely unnecessary.